src/main/java/core/basesyntax/dao/impl/CommentDaoImpl.java

```diff
@@ -12,21 +14,59 @@ public CommentDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Comment create(Comment entity) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(entity);
+            transaction.commit();
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can`t save comment to DB " + entity);
```

Comment:
```suggestion
            throw new RuntimeException("Can`t save comment to DB " + entity, e);
```

Lets also include info about Exception e(check in all classes)